src/utils/tests.ts

```diff
@@ -0,0 +1,73 @@
+import { telemetry } from '../telemetry';
+import { pid } from './pid';
+import { session } from './session';
+import auth from '../Auth';
+import sinon = require('sinon');
+import { Logger } from '../cli/Logger';
+import { Cli } from '../cli/Cli';
+import { sinonUtil } from './sinonUtil';
+import { spo } from './spo';
+
+let cli: Cli;
+export let log: string[] = [];
+export let loggerLogSpy: sinon.SinonSpy;
+export let loggerLogToStderrSpy: sinon.SinonSpy;
+export let promptOptions: any;
+export const logger: Logger = {
+  log: (msg: string) => log.push(msg),
+  logToStderr: (msg: string) => log.push(msg)
+};
+
+export function centralizedBeforeHook(includeRequestDigest: boolean = false): void {
```

Comment:
Maybe instead of the word `centralized` we could use `common` 🤔

Reply:
I've renamed the functions. What do you think of it now?